### PR TITLE
Added aggregate java examples

### PR DESCRIPTION
**Pull request by fboucquez**
Aggregate java examples. Part of the https://github.com/nemtech/symbol-docs/issues/101 task. 

There is a TODO in the source code. I'm going to improve the SDK so it easier for users to add signed cosignatres to aggregate transactions.

### CODE REVIEW

File: source/guides/aggregate/signing-announced-aggregate-bonded-transactions.rst [hunk 1]
```diff
@@ -76,10 +86,10 @@ If you want to cosign a transaction involving a multisig account, you should be
         :start-after:  /* start block 02 */
         :end-before: /* end block 02 */
 
-    .. viewsource:: ../../resources/examples/typescript/aggregate/CosigningAggregateBondedTransactions.js
-        :language: javascript
-        :start-after:  /* start block 02 */
-        :end-before: /* end block 02 */
+    .. viewsource:: ../../resources/examples/java/src/test/java/symbol/guides/examples/aggregate/CosigningAggregateBondedTransactions.java
```
Comment (segfaultxavi) [inline source/guides/aggregate/signing-announced-aggregate-bonded-transactions.rst:89]: Everywhere else Java examples have been ADDED, except here (and below) where Java examples have REPLACED the Javascript ones. Is this correct?

**Review by segfaultxavi — COMMENTED** [review]
Just one question for now. I'll try to build the example and check that everything's fine once I have my system set up.

File: source/guides/aggregate/signing-announced-aggregate-bonded-transactions.rst (hunk 1 above)
Comment (fboucquez) [reply source/guides/aggregate/signing-announced-aggregate-bonded-transactions.rst:89]: good catch pushed a fix

File: source/guides/aggregate/signing-announced-aggregate-bonded-transactions.rst [hunk 2]
```diff
@@ -7,7 +7,7 @@
     :nocomments:
 
 #######################################
-Cosigning aggregate bonded transactions
+Cosigning aggregate bonded transactionsggregate/CosigningAggregateBonde
```
Comment (segfaultxavi) [inline source/guides/aggregate/signing-announced-aggregate-bonded-transactions.rst:10]: Is this title correct?

**Review by segfaultxavi — CHANGES_REQUESTED** [review]

File: source/guides/aggregate/signing-announced-aggregate-bonded-transactions.rst (hunk 2 above)
Comment (fboucquez) [reply source/guides/aggregate/signing-announced-aggregate-bonded-transactions.rst]: Nop, fixed

**Comment by fboucquez** [comment]
Moved to this PR 
https://github.com/nemtech/symbol-docs/pull/503

**Pull request closed without merging**